asabiris/formatter/jinja/service.py

```diff
@@ -28,8 +30,15 @@ def __init__(self, app, service_name="JinjaService"):
 			self.Variables = {}
 
 		self.Environment = jinja2.Environment()
+		# Inject 'now' function into the Jinja2 template's global namespace
+		self.Environment.globals['now'] = self._get_current_time
 		self._load_variables_from_json()
 
+	def _get_current_time(self):
```

Comment:
1) Name `_jinja2_now` ...
2) It should return `current_time` ... a formatting should be done in Jinja2 template